packages/react-components/react-drawer/library/src/components/OverlayDrawer/OverlayDrawer.types.ts

```diff
@@ -14,8 +14,15 @@ export type OverlayDrawerSlots = {
    * Slot for the root element.
    */
   root: Slot<OverlayDrawerSurfaceProps>;
-
+  /**
+   * For more information refer to the [Motion docs page](/?path=/docs/motion-motion-slot--docs).
```

Comment:
will this link actually work ? lets make those links work from both source code and docs

Reply:
but it will not work from source right, that needs to be fixed.

also wondering why it wont work on staging ? you dont have there origin URL (yet) the rest should be same no matter where it lives no?

Reply:
> also wondering why it wont work on staging ? you dont have there origin URL (yet) the rest should be same no matter where it lives no?

staging url is appended with PR number `/pull/32437` after the baseUrl, which gets overwritten by the relative path to motion doc page. Let's use the absolute full path, that will work as expected everywhere after we merge this

Reply:
sounds good, lets go with full path , ty